API.md

```diff
@@ -980,18 +981,29 @@ Get the current status of the service
 | ----- | ---- | ----------- |
 | status | String | The status of the ledger, *OK* if the service is working |
 
+#### Kafka
+Register a Kafka Consumer
+
+##### HTTP Request
+`POST http://central-ledger/kafka`
+
+##### Response 201 OK
+| Field | Type | Description |
+| ----- | ---- | ----------- |
+| status | String | A kafka consumer has been registered for your DFSP Account  |
+
+
 ##### Request
 ``` http
-GET http://central-ledger/health HTTP/1.1
+POST http://central-ledger/kafka HTTP/1.1
 ```
 
 ##### Response
 ``` http
-HTTP/1.1 200 OK
+HTTP/1.1 201 OK
 {
-  "status": "OK"
-}
-```
+    "accountTopic": "topic-dfsp10-prepare-notification"
```

Comment:
What is an "accountTopic"?

As per our discussion you need to also include the purpose of the consumer either in the URI or in the Body of the POST message. 

How will the Kafka consumer know if this should be a consumer for Notifications or Prepare flow?